### DISCUSSION TITLE
How can i make a SlashCommand with specific option per guild ?

### DISCUSSION THREAD

**Discussion by Tech-User42** [tandemdude/hikari-lightbulb#288] [Q&A] [answered]
Hi,
I got an idea but idk if Discord allow it,

My bot can play radio from internet and i want to give the hability to each guild to have their own radio station.

So the radio station were stored and loaded from a json file on the bot startup and on change with an edit command

So when a guild make `/radio play` they will only see their radio stations

![image](https://user-images.githubusercontent.com/60147737/204161425-405ecfa4-4e4a-41d4-b02c-ab6aa99c739e.png)

**Comment by tandemdude** [comment, marked as answer]
For this you probably want to use option autocomplete in order to suggest the appropriate stations per-guild. See [the docs](https://hikari-lightbulb.readthedocs.io/en/latest/api_references/commands.html#lightbulb.commands.base.CommandLike.autocomplete) for info.

**Reply by Tech-User42** [reply]
Hi thanks for your reply i have use this solution and it works like a charm
![image](https://user-images.githubusercontent.com/60147737/206563874-51c4cf42-cae3-43d8-a046-9460ad8c3eb2.png)

```
@radio_commande.autocomplete("station")
async def auto_test(opt: hikari.AutocompleteInteractionOption, inter: hikari.AutocompleteInteraction):
    Settings = SettingsHandler(inter.guild_id)
    Data = Settings.Load()
    to_return = []
    try:
        for radio in Data["radios"]:
            to_return.append(radio["name"])
    except KeyError:
        to_return = ["Aucune Station Enregistrée !"]
    return to_return
```
